src/pages/products/index.tsx

```diff
@@ -1,59 +1,102 @@
-import { useContext, useEffect, useState } from 'react'
-import { useParams } from 'react-router-dom'
+import { useEffect, useMemo, useState } from 'react'
 
+import { TTarget } from './types'
+import { deleteProduct, getProducts, updateProduct } from 'contracts/products'
 import { IProduct } from 'interfaces/IProduct'
-import { UserContext } from 'contexts/userContext'
 import { EditableText } from 'components/editableText'
-import { findProduct, updateProduct } from 'contracts/products'
+import { Edit } from './svgIcons/edit'
+import { Delete } from './svgIcons/delete'
 import styles from './styles.css'
+import { Link } from 'react-router-dom'
+import { WarningWindow } from 'components/warningWindow'
 
 export const Products = () => {
-  const [product, setProduct] = useState<IProduct>({} as IProduct)
-  const { title } = useParams()
-  const { isAuthenticated } = useContext(UserContext)
-  const [description, setDescription] = useState('')
+  const [value, setValue] = useState('')
+  const [products, setProducts] = useState<IProduct[]>([])
+  const [editIndex, setEditIndex] = useState(-1)
+  const [id, setId] = useState<number | undefined>(undefined)
+  const productsWithImages = products.filter(product => value.length >= 2 ? product.title.includes(value) : product.image)
```

Comment:
I don't understand why you show only `product.image`. How I can see other products?